lib/OpenQA/Worker/Engines/isotovideo.pm

```diff
@@ -137,8 +137,8 @@ sub cache_assets {
         if ($asset_request->enqueue) {
             log_debug("Downloading " . $asset_uri . " - request sent to Cache Service.", channels => 'autoinst');
             update_setup_status and sleep 5 until $asset_request->processed;
-            log_debug("Download of " . $asset_uri . " processed", channels => 'autoinst');
-            log_debug($asset_request->output,                     channels => 'autoinst');
+            my $msg = "Download of " . $asset_uri . " processed";
+            $msg .= ": $output" if (my $output = $assert_request->output);
```

Comment:
This is broken Perl. The variable declaration needs to come first.

Reply:
I was also surprise that this worked but suspected some Perl magic going on here :-)

Btw, you could also use interpolation for the concatenation above.

Reply:
Yes, interpolation would look better.
```perl
my $msg = "Download of $asset_uri processed";
if (my $output = $assert_request->output) { $msg .= ": $output" }
```

Reply:
hm, ok. worked in `perl -de1` but I guess the variable scope behaves different then.

Changed. PR updated.